lib/auto-login.js

```diff
@@ -178,6 +168,41 @@ module.exports = function setupAutoLogin(dependencies, params, callback) {
 
   }
 
+  function trySetMusicoinProfileURL(user, uid, callback) {
+  	
+  	logger.info({ method: 'trySetMusicoinProfileURL', input: user, type: 'start' });
+
+  	let namespace = 'ns:custom_fields';
+
+  	async.waterfall([(done) => {
+  		
+  		db.getObject(createObjectKey(uid, namespace), (error, data) => {
+      	done(null, uid, data);
+      });
+
+  	}, (uid, data, done) => {
+
+  		if(data && data.musicoinprofile) {
+  			return done(null, uid);
+  		}
+
+  		let customFields = {musicoinprofile: `${appURL}/artist/${user.profileAddress}`};
```

Comment:
does profile address just provide for artist?  or this is valid for all users?

Reply:
For all users, I believe. This is better answered by @isaac or @Varunram

Reply:
api can check a user is artist or not? if we want to valid for just artists. we can check like this
`if(user.isArtist) {  
     let customFields = {musicoinprofile: `${appURL}/artist/${user.profileAddress}`};
}else callback();
`

Reply:
@immartian , @Varunram @livey0u we need to decide about this.

Reply:
we can support all users, no need to restrict to artists only(or verified artist)

Reply:
okay, we can merge.